### PR TITLE
Created useFakeProgress hook

### PR DESCRIPTION
**Pull request by mariush2**
- [AB#465682](https://dev.azure.com/Equinor/4319082b-29ba-4111-a20b-c04f55c7e7c6/_workitems/edit/465682)
- [AB#465683](https://dev.azure.com/Equinor/4319082b-29ba-4111-a20b-c04f55c7e7c6/_workitems/edit/465683)
- [AB#465684](https://dev.azure.com/Equinor/4319082b-29ba-4111-a20b-c04f55c7e7c6/_workitems/edit/465684)

### CODE REVIEW

File: src/components/Feedback/SelectField/ChangingField.tsx [hunk 1]
```diff
@@ -53,33 +54,9 @@ const ChangingField: FC<ChangingFieldProps> = ({
...
-  }, []);
-
-  useEffect(() => {
-    if (finished) {
-      setTimeout(() => {
-        onChangedField();
-      }, 4000);
-    }
-  }, [finished, onChangedField]);
+  console.log('finished', finished);
```
Comment (aslakihle) [inline src/components/Feedback/SelectField/ChangingField.tsx:59]: Console log 🤫

File: src/hooks/useFakeProgress.test.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    if (changeFinishMs) {
+      setTimeout(() => {
+        setFinishMs((prev) => prev + 10);
+      }, 100);
+    }
+  }, [changeFinishMs]);
+
+  return <div>{progress}</div>;
+}
+
```
Comment (aslakihle) [inline src/hooks/useFakeProgress.test.tsx:46]: I think there should be some more tests checking if onDone was called, and maybe if it lasted the correct amount of time.

**Review by aslakihle — CHANGES_REQUESTED** [review]

File: src/hooks/useFakeProgress.test.tsx (hunk 2 above)
Comment (mariush2) [reply src/hooks/useFakeProgress.test.tsx]: I can add a test for that 😄

Comment (aslakihle) [reply src/hooks/useFakeProgress.test.tsx]: 👌

**Review by aslakihle — APPROVED** [review]

**Pull request merged**